projects/storefrontlib/src/cms-components/organization/permissions/permission-form/permission-form.component.ts

```diff
@@ -33,6 +33,7 @@ export class PermissionFormComponent implements OnInit {
   businessUnits$: Observable<B2BUnitNode[]>;
   currencies$: Observable<Currency[]>;
   permissionTypes$: Observable<OrderApprovalPermissionType[]>;
+  selectedPermissionType: OrderApprovalPermissionType['code'];
```

Comment:
If we don't use enum here, the type of selectedPermissionType is just string.
But we needn't that field usage in ngModel and ngIf anymore, so you can remove that

Reply:
done